client/components/Theme.jsx

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import { ThemeProvider } from 'styled-components';
```

Comment:
Sometimes you use destructuring like here and other times you don't (noticed it with extend React.component in the reviewswidget file.)

Reply:
Most of the time I used the default export (like styled), but to import the named exports like ThemeProvider you need destructuring: https://styled-components.com/docs/api.